sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/PodInstanceRequirement.java

```diff
@@ -19,6 +17,8 @@
     private final Map<String, String> environment;
     private final RecoveryType recoveryType;
 
+    private static final Logger logger = LoggerFactory.getLogger(PodInstanceRequirement.class);
```

Comment:
Logger should be non-static (since it doesn't need to be accessed from any static methods)